test/unit/test/permissions.cpp

```diff
@@ -0,0 +1,100 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2021, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2021, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+#include <gtest/gtest.h>
+
+#include <seqan3/test/file_access.hpp>
+#include <seqan3/test/tmp_directory.hpp>
+#include <seqan3/test/tmp_filename.hpp>
```

Comment:
This test is cyclic;

* seqan3/test/file_access.hpp
* tested by file_access_test.cpp
* uses seqan3/test/tmp_directory.hpp
* tested by tmp_directory_test.cpp
* uses seqan3/test/file_access.hpp

Can we write this test in a way that does not need seqan3/test/tmp_directory.hpp and seqan3/test/tmp_filename.hpp?

Reply:
I could, e.g., use the current working directory and skip the tests if I do not have permissions to write there

Reply:
Sounds like a great idea.